scripts/composer/ScriptHandler.php

```diff
@@ -41,7 +41,7 @@ public static function createRequiredFiles(Event $event) {
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
       $settings['config_directories'] = [
-        CONFIG_SYNC_DIRECTORY => (object) [
+        'config_sync_directory' => (object) [
           'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config/sync', $drupalRoot),
```

Comment:
I'm not sure if this is happily ignored and uses what *we* use as default (e.g: /config)
```suggestion
          'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config', $drupalRoot),
```